resources/language/resource.language.nl_nl/strings.po

```diff
@@ -511,8 +511,16 @@ msgid "Update add-on repositories"
 msgstr "Add-on repositories worden bijgewerkt"
 
 msgctxt "#30451"
-msgid "If a VRT NU add-on update is available, it will now be updated and you will be notified.\nMake sure you have [I]Auto-update[/I] enabled in Kodi and the VRT NU add-on."
-msgstr "Als een VRT NU update beschikbaar is, zal deze nu worden geïnstalleerd en krijg je hiervan een melding.\nControleer wel dat [I]Automatische update[/I] is ingesteld voor Kodi en de VRT NU add-on."
+msgid "If a VRT NU add-on update is available, it should now be updating and you will be notified.\nMake sure you have [I]Auto-update[/I] enabled for the VRT NU add-on."
+msgstr "Als een VRT NU update beschikbaar is, zal deze nu geïnstalleerd worden en krijg je hiervan een melding.\nControleer wel dat [I]Automatische update[/I] is ingesteld voor de VRT NU add-on."
+
+msgctxt "#30452"
+msgid "Automatic updates are disabled"
+msgstr "Automatische updates zijn niet geactiveerd"
+
+msgctxt "#30453"
+msgid "Kodi is configured to not update add-ons automatically, so you will not get VRT NU updates unless you fix this setting.\nWe will open de Kodi settings, please change this setting."
+msgstr "Kodi is ingesteld om niet automatisch add-ons bij te werken, dus je zal geen VRT NU updates ontvangen tenzij je deze instelling aanpast.\nWe openen nu de Kodi settings zodat je dit kan wijzigen."
```

Comment:
Andere woordvolgorde: Kodi is ingesteld om  add-ons niet automatisch bij te werken

Reply:
Kodi settings = Kodi-instellingen